actions/pull-request/check-commits-unverified/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM alpine
+
+RUN apk add \
+    bash \
+    curl \
+    git \
+    jq \
+  && rm -rf /var/cache/apk/*
+
+RUN mkdir /lib64 && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
+
+ARG gh_version=0.11.1
+RUN mkdir -p "/bin" \
+      && export PATH="${PATH}:/bin" \
+      && curl "https://github.com/cli/cli/releases/download/v${gh_version}/gh_${gh_version}_linux_amd64.tar.gz" \
+        --silent \
+        --location \
+        --output "/tmp/gh.tgz" \
+      && tar xzf "/tmp/gh.tgz" --strip-components 1 -C / \
+      && chmod +x "/bin/gh"
```

Comment:
Doesn't look like the entrypoint uses `gh`, and so this could probably be removed.